core/crypto/aes-gcm-sw.c

```diff
@@ -43,11 +43,11 @@ void internal_aes_gcm_ghash_update(struct internal_aes_gcm_state *state,
 	if (head)
 		ghash_update_block(state, head);
 
-	if (data) {
+	if (data)
 		for (n = 0; n < num_blocks; n++)
 			ghash_update_block(state,
-					   (const uint8_t *)data + n * TEE_AES_BLOCK_SIZE);
-	}
+					   (const uint8_t *)data +
+					   n * TEE_AES_BLOCK_SIZE);
```

Comment:
This change belongs to commit "core: refactoring to avoid possible NULL_PTR arithmetics".